Tests/AppTests/PipelineTests.swift

```diff
@@ -165,7 +162,7 @@ class PipelineTests: AppTestCase {
         do {  // validate - only new package moves to .ingestion stage
             let packages = try Package.query(on: app.db).sort(\.$url).all().wait()
             XCTAssertEqual(packages.map(\.url), ["1", "3", "4"].asGithubUrls)
-            XCTAssertEqual(packages.map(\.status), [.ok, .ok, .ok])
+            XCTAssertEqual(packages.map(\.status), [.ok, .ok, .none])
```

Comment:
The full pipeline test `test_processing_pipeline` shows nicely how the `status` field progresses and ends up being `.ok` eventually.